src/App.js

```diff
@@ -43,6 +44,8 @@ class App extends React.Component {
 					isOpen={settingsDrawerIsOpen}
 					onClose={this.toggleSettingsDrawer}
 				>This is the settings drawer.</SettingsDrawer>
+
+				<div className="overlay"></div>
```

Comment:
Not to be too cheeky, but you could be super efficient and swap this out to be an ``:after`` component on the ``.SettingsDrawer`` 😎

Reply:
Turns out I was being too cheeky, using ``after`` won't work due to a conflict with ``.SettingsDrawer`` having ``transform``, which would reset the anchor used for ``position: fixed;``. Please disregard this.

Reply:
You was too cheeky and it backfired